### ISSUE TITLE
Blocking system updates does not work at WebOS 6.2

### ISSUE THREAD

**Issue by Bodenseematze** [webosbrew/webos-homebrew-channel#89]
I switched on the flag to block system updates.
But the TV still told me that an update is available and should be installed - every time I switched it on...

LG OLED48C17LB with WebOS 6.2

**Comment by Informatic** [comment]
Is that when the failsafe mode is triggered (then that is expected, since with failsafe mode we bail out early and do not block updates), or during normal bootup when the notification doesn't pop up?

Can you please share your version information generated by this command:
```sh
grep -h -E '"(hardware_id|core_os_release|product_id|webos_manufacturing_version|board_type)"' /var/run/nyx/*
```

**Comment by Bodenseematze** [comment]
> Is that when the failsafe mode is triggered (then that is expected, since with failsafe mode we bail out early and do not block updates), or during normal bootup when the notification doesn't pop up?
Both - it was when the failsave flag was triggered but after removing and rebooting the message came again...


> 
> Can you please share your version information generated by this command:
> 
> ```shell
> grep -h -E '"(hardware_id|core_os_release|product_id|webos_manufacturing_version|board_type)"' /var/run/nyx/*
> ```

`    "board_type": "O20_DVB",`
`    "hardware_id": "HE_DTV_W21O_AFABATAA",`
`    "product_id": "OLED48C17LB",`
`    "core_os_release": "6.2.0-35",`
`    "webos_manufacturing_version": "03.21.20",`

**Comment by throwaway96** [comment]
Closed in favor of #96.
